tests/baselines/reference/jsxElementType.errors.txt

```diff
@@ -197,4 +197,15 @@ tests/cases/compiler/jsxElementType.tsx(99,2): error TS2304: Cannot find name 'U
     <Unresolved foo="abc" />;
      ~~~~~~~~~~
 !!! error TS2304: Cannot find name 'Unresolved'.
+    
+    // regression test for intrinsic containing `:`
+    declare global {
+      namespace JSX {
+        interface IntrinsicElements {
+          ['fbt:enum']: { knownProp: string };

```

Comment:
Maybe this needs to be declared another way now that https://github.com/microsoft/TypeScript/pull/47356/ has landed? But even if we drop support for that, the error message currently produced is not helpful. It silently treats `<fbt:enum />` as any.